packages/components/Button/src/Button.styling.android.ts

```diff
@@ -0,0 +1,116 @@
+import { buttonName, ButtonCoreTokens, ButtonTokens, ButtonSlotProps, ButtonProps, ButtonSize } from './Button.types';
+import { Theme, UseStylingOptions, buildProps } from '@fluentui-react-native/framework';
+import { borderStyles, layoutStyles, fontStyles, FontTokens } from '@fluentui-react-native/tokens';
+import { defaultButtonTokens } from './ButtonTokens';
+import { defaultButtonColorTokens } from './ButtonColorTokens';
+import { Platform, ColorValue } from 'react-native';
+import { getTextMarginAdjustment } from '@fluentui-react-native/styling-utils';
+import { defaultButtonFontTokens } from './ButtonFontTokens';
+
+export const buttonCoreStates: (keyof ButtonCoreTokens)[] = ['hovered', 'focused', 'pressed', 'disabled', 'hasContent', 'hasIconBefore'];
+
+export const buttonStates: (keyof ButtonTokens)[] = [
+  'block',
+  'primary',
+  'subtle',
+  'hovered',
+  'small',
+  'medium',
+  'large',
+  'hasContent',
+  'hasIconAfter',
+  'hasIconBefore',
+  'rounded',
+  'circular',
+  'square',
+  'focused',
+  'pressed',
+  'disabled',
+];
+
+export const stylingSettings: UseStylingOptions<ButtonProps, ButtonSlotProps, ButtonTokens> = {
```

Comment:
I would just fork the functions that are different. You don't need to copy things that don't change - if they don't exist in the forked file they'll get picked up from the main file correctly.